app/views/games/finished.html.erb

```diff
@@ -9,7 +9,7 @@
       </div>
     </div>
     <div class="too_late">
-      <% if @game.players.include?(current_player) && current_player.won == false %>
+      <% if @game.players.include?(current_player) && current_player.won %>
```

Comment:
Exactly, except that I wouldn't add `Game#loser_is?` until we actually need it.

On the same note, we could add a `Game#winner` and `Game#loser`, though (maybe through associations, e.g. `has_one :winner, -> { merge(Player.winners) }, class_name: "Player"`, or trough association extensions).